### PR TITLE
Local Time display

### PR DESCRIPTION
**Pull request by saikeshari**

### CODE REVIEW

**Comment by saikeshari** [comment]
@grace-omotoso Fixed the bug. Please review

File: src/components/WeatherCard.js [hunk 1]
```diff
@@ -3,22 +3,47 @@ import {Droplet, Wind, Thermometer} from 'react-feather';
...
+  
+    if (time.length > 1) { // If time format correct
+      time = time.slice (1);  // Remove full string match value
+      time[5] = +time[0] < 12 ? ' AM' : ' PM'; // Set AM/PM
+      time[0] = +time[0] % 12 || 12; // Adjust hours
+    }
+    return time.join (''); // return adjusted time or original string
+  }
+  
+  // console.log(tConvert (datee));
```
Comment (grace-omotoso) [inline src/components/WeatherCard.js]: remove commented code

File: src/components/WeatherCard.js [hunk 2]
```diff
@@ -3,22 +3,47 @@ import {Droplet, Wind, Thermometer} from 'react-feather';
...
+
+  let displayTime = tConvert(datee);
+
   let temp=new Date(city.dt);
   let temp1=temp.getMinutes();
 
-  console.log(temp1);
+  // console.log(temp1);
```
Comment (grace-omotoso) [inline src/components/WeatherCard.js:36]: remove commented code

**Comment by saikeshari** [comment]
Removed the commented code @grace-omotoso

**Review by grace-omotoso — APPROVED** [review]

**Pull request merged**